django/db/models/query.py

```diff
@@ -473,6 +473,52 @@ def bulk_create(self, objs, batch_size=None, ignore_conflicts=False):
 
         return objs
 
+    def bulk_update(self, objs, fields, batch_size=None):
+        """
+        Update the given fields in each of the given objects in the database.
+        (Doesn't call save() on each of the instances.)
+        """
+        if batch_size is not None and batch_size < 0:
+            raise ValueError('Batch size must be a positive integer.')
+        if not fields:
+            raise ValueError('Specific field names must be given to bulk_update().')
+        objs = tuple(objs)
+        if not all(obj.pk for obj in objs):
+            raise ValueError('All objects must have a primary key set.')
+        fields = [self.model._meta.get_field(name) for name in fields]
+        if any(not f.concrete or (f.is_relation and f.many_to_many) for f in fields):
```

Comment:
Is there any cases of `not f.is_relation and f.many_to_many`? If not then `f.many_to_many` should be sufficient.